packages/protocol/test-sol/stability/Reserve.t.sol

```diff
@@ -126,6 +147,24 @@ contract ReserveTest_initAndSetters is ReserveTest {
     reserve.setDailySpendingRatio(100);
   }
 
+  function test_dailySpendingRatioForErc20Tokens() public {
```

Comment:
Split this test into separate functions that describe the method & states being tested. Checking multiple unrelated things in a single test makes debugging failures in the CI pipeline a little bit more complex. If you had these as separate tests that are named descriptively, anyone can determine the failure reason at a glance.

The naming convention we should be using for our tests is:
`test_MethodName_StateUnderTest_ExpectedBehaviour`

If you follow this, you could have test functions that look like this:
 - `test_setDailySpendingRatio_whenSenderIsNotOwner_shouldRevert`
 - `test_setDailySpendingRatio_whenRatioIsLargerThanOne_shouldRevert`
 - `test_setDailySpendingRatio_whenSenderIsOwner_shouldUpdateAndEmit`